### PR TITLE
Fix Standalone GC table format

### PR DESCRIPTION
**Pull request by sebastienros**
## Summary

The table markdown is badly formatted and doesn't render as a table right now:
https://learn.microsoft.com/en-us/dotnet/core/runtime-config/garbage-collector#standalone-gc

image

### CODE REVIEW

File: docs/core/runtime-config/garbage-collector.md [hunk 1]
```diff
@@ -557,6 +557,7 @@ Example:
 ## Standalone GC
 
 - Specifies the name of a standalone GC DLL that the runtime loads in place of the one it has in the main runtime DLL (coreclr.dll). This DLL needs to reside in the same directory as *coreclr.dll*
```
Comment (gewarren) [inline docs/core/runtime-config/garbage-collector.md:559]:
```suggestion
- Specifies the name of a standalone GC DLL that the runtime loads in place of the one it has in the main runtime DLL (coreclr.dll). This DLL needs to reside in the same directory as *coreclr.dll*.
```

**Review by gewarren — APPROVED** [review]
Thank you @sebastienros!

File: docs/core/runtime-config/garbage-collector.md (hunk 1 above)
Comment (sebastienros) [reply docs/core/runtime-config/garbage-collector.md:559]: Maybe you were trying to be polite, but I would not have minded if you had updated the PR directly in GH. (assuming you know you can do it, even on forks)

Comment (gewarren) [reply docs/core/runtime-config/garbage-collector.md]: Yes, I wanted to be polite but would have eventually committed it if necessary. Thanks!

**Pull request merged**